frontend/catalyst/jax_primitives.py

```diff
@@ -677,6 +680,81 @@ def _qinsert_lowering(
     return InsertOp(qreg_type, qreg_old, qubit, idx=qubit_idx).results
 
 
+#
+# gphase
+#
+@gphase_p.def_abstract_eval
+def _gphase_abstract_eval(
+    *qubits_or_params, op=None, qubits_len: int = 0, params_len: int = 0, ctrl_len: int = 0
+):
+    # The signature here is: (using * to denote zero or more)
+    # qubits*, params*, ctrl_qubits*, ctrl_values*
```

Comment:
We can remove the `qubits_len` as well from here and the functions below (and the checking of the non-existent target qubits).

Reply:
Let's leave that as a TODO for Tuesday.